### DISCUSSION TITLE
Add. Светлая версия темы для шаблона Dark

### DISCUSSION THREAD

**Discussion by azarro** [madromas/esoTalk-v3#5] [Q&A] [answered]
А есть в планах, добавить иконку переключения на светлую тему. Как на сайте **libarea.ru**

**Comment by madromas** [comment]
Не знаю пока. Надо посмотреть. Как видишь было много ошибок, как то не до этого было. Но это сделать не так уж и сложно.

**Reply by madromas** [reply]
Очень много придется подделать под светлую версию, займусь как нибудь.

**Reply by madromas** [reply, marked as answer]
Done!
